src/measurements/measurements.ts

```diff
@@ -39,7 +37,10 @@ export function distance(from: Point, to: Point): number {
  * @returns distance in meters
  */
 export function geodesicDistance(from: Point, to: Point): number | undefined {
-  const r = geod.Inverse(from.coordinates.lat, from.coordinates.lon, to.coordinates.lat, to.coordinates.lon);
+  const [fromLon, fromLat] = from.coordinates;
+  const [toLon, toLat] = to.coordinates;
+
+  const r = geod.Inverse(fromLat, fromLon, toLat, toLon);
   return r.s12;
```

Comment:
What is this? I understand it's a distance, but why is it `s12`?

Reply:
it's the package's notation for a distance between point1 and point2

Reply:
Ahhh so readable...